ui/app/src/components/DashboardCard/DashboardCard.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Button, Stack, Typography } from '@mui/material';
+import ViewDashboardIcon from 'mdi-material-ui/ViewDashboard';
+import { DashboardResource } from '@perses-dev/core';
+import { dashboardDisplayName } from '@perses-dev/core/dist/utils/text';
+import FolderPoundIcon from 'mdi-material-ui/FolderPound';
+import { useNavigate } from 'react-router-dom';
+
+interface DashboardCardProps {
+  dashboard: DashboardResource;
+}
+
+export function DashboardCard(props: DashboardCardProps) {
+  const naviguate = useNavigate();
+
+  return (
+    <Button
```

Comment:
The color of these buttons seem a little too bright. Maybe we should use the same color as the top navbar?

Reply:
It will look like this, is it better?
![image](https://user-images.githubusercontent.com/5657041/233005823-0c9ae810-0294-4d95-a7bc-8224f408b629.png)
Using the same background color (`theme.palette.designSystem.blue[700]`)